packages/strapi-admin/admin/src/containers/NewNotification/Notification/index.js

```diff
@@ -100,8 +108,12 @@ const Notification = ({ notification }) => {
                       >
                         {formattedMessage(link.label)}
                       </Text>
-                      <Padded left size="xs" />
-                      <LinkArrow />
+                      {link.target === "_blank" && (
+                        <>
+                          <Padded left size="xs" />
+                          <LinkArrow />
+                        </>
```

Comment:
Can you wrap the LinkArrow with the Padded component and remove the empty tags please 😉 .

```js
<Padded left size="xs">
   <LinkArrow />
</Padded>
```

Reply:
Whaha, true my bad, done!